finality-aleph/src/party/mod.rs

```diff
@@ -224,33 +59,21 @@ async fn get_node_index(
         .map(|id| id.into())
 }
 
-fn run_justification_handler<B, V, RB, C, S, SI, F>(
-    handler: JustificationHandler<B, V, RB, C, S, SI, F>,
-    spawn_handle: &crate::SpawnHandle,
-    import_justification_rx: mpsc::UnboundedReceiver<JustificationNotification<B>>,
-) -> mpsc::UnboundedSender<JustificationNotification<B>>
-where
-    C: HeaderBackend<B> + Send + Sync + 'static,
-    B: Block,
-    RB: RequestBlocks<B> + 'static,
-    V: Verifier<B> + Send + 'static,
-    S: JustificationRequestScheduler + Send + 'static,
-    SI: SessionInfoProvider<B, V> + Send + Sync + 'static,
-    F: BlockFinalizer<B> + Send + 'static,
-{
-    let (authority_justification_tx, authority_justification_rx) = mpsc::unbounded();
-
-    debug!(target: "afa", "JustificationHandler started");
-    spawn_handle.spawn("aleph/justification_handler", async move {
-        handler
-            .run(authority_justification_rx, import_justification_rx)
-            .await;
-    });
-
-    authority_justification_tx
+pub(crate) struct ConsensusPartyParams<B: Block, SC, C, RB> {
```

Comment:
actually this struct is used in one place and we gain at most nothing; I would drop it as it is unnecessary and effectively adds more code

Reply:
we gain one less warning from the clippy :open_hands: